content/_index.html

```diff
@@ -50,10 +50,15 @@ <h1>Participating</h1>
 
     <h2>Upcoming meetings</h2>
     <ul>
-        <li>2024-03-05 - <a href="/post/2024-02-08_preparation">Kickoff preparation meeting</a></li>
         <li>2024-05-07 - Kickoff meeting</li>
     </ul>
 
+    <h2>All Posts</h2>
+    <ul>
+        <li>2024-03-12 - <a href="/post/2024-03-12_kickoff-preparation">Summary of the preparation meeting</a></li>
+        <li>2024-03-05 - <a href="/post/2024-02-08_preparation">Kickoff preparation meeting</a></li>
+    </ul>
+
```

Comment:
With Hugo, it should be possible to automatically list all posts. I don't know how to do it though

Reply:
#3